### PR TITLE
Adds argument for HELO domain

### PR DESCRIPTION
**Pull request by viardant**
Avoids `ERROR 504 Helo command rejected: need fully-qualified hostname;` in servers that require FQDN HELO.

### CODE REVIEW

File: smtp-user-enum [hunk 1]
```diff
@@ -433,6 +434,15 @@ Default: Nothing wrapped""",
...
         help="MAIL FROM email address. Only used in RCPT mode" + "\nDefault: " + DEFAULT_MAIL_FROM,
     )
+    parser.add_argument(
+        "-l",
+        "--helo",
+        metavar="addr",
+        required=False,
+        default=DEFAULT_HELO,
+        type=str,
+        help="Domain name of sending host used in HELO command."
```
Comment (cytopia) [inline smtp-user-enum]: Can you add the `+ "\nDefault: " + DEFAULT_HELO` as it was done in line 435

File: README.md [hunk 2]
```diff
@@ -422,12 +423,12 @@ timed out
...
 
 ```bash
-$ smtp-user-enum -V -m RCPT -U /usr/share/wordlists/metasploit/unix_users.txt mail.example.tld 25
+$ smtp-user-enum -V -m RCPT -U /usr/share/wordlists/metasploit/unix_users.txt -l mydomain.com mail.example.tld 25
 Connecting to mail.example.tld 25 ...
 [1/4] Connecting to mail.example.tld:25 ...
 [1/4] Waiting for banner ...
 220 beta SMTP Server (JAMES SMTP Server 2.3.2) ready Wed, 22 Jan 2020 16:10:10 -0500 (EST)
-[1/4] Sending greeting: HELO test
+[1/4] Sending greeting: HELO mydomain.com
```
Comment (cytopia) [inline README.md]:
```suggestion
[1/4] Sending greeting: HELO mydomain.com
```

File: README.md [hunk 3]
```diff
@@ -287,7 +288,7 @@ $ nc mail.example.tld 25
 ```
 ```
 220 mail.example.tld ESMTP Sendmail 8.12.8/8.12.8; Thu, 23 Jan 2020 16:03:22 +0200
-HELO test
+HELO changeme
```
Comment (cytopia) [inline README.md]:
```suggestion
HELO changeme
```

File: README.md [hunk 4]
```diff
@@ -218,7 +219,7 @@ $ nc mail.example.tld 25
...
```
Comment (cytopia) [inline README.md]:
```suggestion
HELO changeme
```

**Review by cytopia — COMMENTED** [review]
Thanks for the PR. A few minors to adjust.

File: smtp-user-enum (hunk 1 above)
Comment (viardant) [reply smtp-user-enum]: Sure thing! Commited few seconds ago

**Review by cytopia — APPROVED** [review]
Thanks! :+1:

**Comment by cytopia** [comment]
@viardant Published: https://pypi.org/project/smtp-user-enum/0.6.0/

**Pull request merged**